cmd/ctr/commands/run/run.go

```diff
@@ -93,7 +93,7 @@ var Command = cli.Command{
 	Flags: append([]cli.Flag{
 		cli.BoolFlag{
 			Name:  "rm",
-			Usage: "remove the container after running",
+			Usage: "remove the container after running, cannot be specified with --detach together",
```

Comment:
to keep these usage strings from getting longer, could we simplify the added text to: `, cannot be used with --detach`?

Reply:
Yes, `cannot be used with --detach` looks better, thanks